### ISSUE TITLE
[APP] Timeout bug?

### ISSUE THREAD

**Issue by JaafarRammal** [SOMAS2021/SOMAS2021#217]
```go
//channel and goroutine used for timeout
c1 := make(chan string, 1)

go func() {
	runNewSimulation(parameters, logFolderName)
	c1 <- "Simulation Finished"
}()

// Listen on our channel AND a timeout channel - which ever happens first.
select {
case <-c1:
	log.Info("Simulation Finished Successfully")
case <-time.After(time.Duration(parameters.SimTimeoutSeconds) * time.Second):
	log.Fatal("Simulation Timeout")
}
```

We never stop `c1` so it is still running in the background

**Comment by VFjr** [comment]
Hmm, according to this we don't really have to bother with it as it will be automatically garbage collected:

https://stackoverflow.com/questions/8593645/is-it-ok-to-leave-a-channel-open

**Comment by VFjr** [comment]
Fixed in #222 (hopefully)
